### ISSUE TITLE
Podsumowanie kosztów oferty, który endpoint pozwala na pobieranie danych o kosztach oferty?

### ISSUE THREAD

**Issue by grzesieksw** [allegro/allegro-api#8811]
Chciałbym pobrać poprzez API podsumowanie kosztów oferty, i moje główne pytanie to, który endpoint za to odpowiada?
Na stonie allegro, mogę łatwo wejść w **Mój asortyment** -> i wybierając konkretny produkt kliknąć na **Zobacz podsumowanie kosztów**

Zauważyłem, że można pobierać ciekawe dane używając endpointu: **/billing/billing-entries** i podając konkretny **offer.id**, dane są tam szczegółowo rozbite (co samo w sobie nie jest problemem) jednak wygląda że często dane tam nie do końca są kompletne, dotyczy to w szczególności opłat za przesyłki kurierów itp, wchodząc na stronę allegro i wchodząc właśnie w podsumowanie kosztów oferty, opłaty za przesyłkę są widoczne, a w przypadku API często nie są widoczne (być może przesyłka musi być potwierdzona aby tam się pokazał wpis?)

Dodatkowo zauważyłem, że gdy już jest jakaś opłata billingowa to ma ona wyłącznie powiązanie z **oder.id**, a nie z **offer.id** które jest wtedy puste, co utrudnia filtrowanie po offer.id. Teoretycznie może mieć to sens, jednak używając strony allegro ładnie po offer.id można wyciągnąć wszystkie koszty.

Czy może istnieje inny endpoint, którego nie zauważyłem?

**Comment by AureliuszBrussy** [comment]
Jest to jedyny endpoint do pobierania informacji o opłatach. Dzięki [GET /billing/billing-types](https://developer.allegro.pl/documentation#operation/getBillingTypes) możesz też sprawdzić wszystkie typy opłat z opisem. 

Wpisy powinny powinny być zbieżne z tym co widzisz w zakładce Rozliczenia z Allegro. Zdajemy sobie sprawę jednak, że może brakować dodatkowych szczegółów (np. kwestia poruszona [w wątku](https://github.com/allegro/allegro-api/issues/8303)).

Jeżeli widzisz jakieś pole do poprawy - daj znać na podstawie konkretnego wpisu gdzie widzisz go w zakładce, a nie widzisz go w odpowiedzi z endpointu, lub gdzie widzisz niepełne informacje dot. np. id oferty. Możesz przekazać dane przez [formularz kontatkowy](https://allegro.pl/pomoc/kontakt) (przez e-maila). W treści przekaż, że sprawa dotyczy zgłoszenia #8811 na GitHubie.

**Comment by AureliuszBrussy** [comment]
Otrzymaliśmy zgłoszenie. Wysłałeś zrzut ekranu z zakładki ["Podusumowanie kosztów"](https://allegro.pl/moje-allegro/sprzedaz/podsumowanie-kosztow). Endpoint /billing/billing-entries odzwierciedla działanie zakładki "[Rozliczenia z Allegro](https://allegro.pl/moje-allegro/moje-konto/rachunki/rozliczenia-z-allegro)". Czy widzisz jakieś niezgodności pomiędzy tą zakładką, a wpisami otrzymanymi w odpowiedzi na GET /billing/billing-entries? Jeżeli tak - podeślij w wątku mailowym przykłady.

**Comment by grzesieksw** [comment]
Zgadza się zakładka rozliczenia z allegro zawiera również jeden wpis zgodny z danymi które otrzymuje z endpointu: 
**/billing/billing-entries.**

Jednak tak jak w pierwszej wiadomości napisałem:
> Na stonie allegro, mogę łatwo wejść w Mój asortyment -> i wybierając konkretny produkt kliknąć na Zobacz podsumowanie kosztów

Prostu ujmując biznes chciałby uzyskać informacje, ile faktycznie kosztował go dany produkt, aby wiedzieć czy jest sprzedawany z odpowiednią marżowością, czy mogę uzyskać te dane wykorzystując API allegro?

**Comment by AureliuszBrussy** [comment]
Nie, przez API nie uzyskasz dokładnie tego samego efektu jak w zakładce "Zobacz podsumowanie kosztów"
